staging/src/k8s.io/csi-translation-lib/plugins/azure_disk_test.go

```diff
@@ -269,7 +269,7 @@ func TestTranslateInTreeStorageClassToCSI(t *testing.T) {
 		},
 		{
 			name:       "some translated topology",
-			options:    NewStorageClass(map[string]string{}, generateToplogySelectors(v1.LabelFailureDomainBetaZone, []string{"foo"})),
+			options:    NewStorageClass(map[string]string{}, generateToplogySelectors(v1.LabelTopologyZone, []string{"foo"})),
```

Comment:
Can we keep the test case for the beta labels too? This function needs to support both beta and GA labels.